src/config/machines/transition-patterns.js

```diff
@@ -0,0 +1,20 @@
+// src/config/machines/transition-patterns.js
+
+/**
+ * Creates standard transitions for a page in the journey
+ * @param {string} backTarget - The state to go back to
+ * @param {string} nextTarget - The state to go forward to
+ * @returns {object} Standard transition configuration
+ */
+export const createTransitions = (backTarget, nextTarget) => ({
+  on: {
+    BACK: {
+      target: backTarget,
+      actions: ['updateCurrentPageId']
+    },
+    NEXT: {
+      target: nextTarget,
+      actions: ['trackPageCompletion', 'updateCurrentPageId', 'updateAnswers']
```

Comment:
This is assuming that all states require an answer update and page completion. Possibly, actions as defaulted arguments would be a good idea here.